### ISSUE TITLE
Loading empty track generates error (nullPointerException) and arguably should not.

### ISSUE THREAD

**Issue by malcook** [igvteam/igv#754]
Loading sessions referring to empty files reports null pointer exception for each empty file (at least for .narrowpeak).

A better user experience would be to warn of empty files.

**Comment by malcook** [comment]
@jrobinso - might this have fallen off your radar?  

For further motivation of its value, I offer:

It would be valuable to be able to show empty tracks, at least in experiments in which I am using IGV for EDA where an effect completely disappears genome-wide over a time-series.

**Comment by jrobinso** [comment]
No, my radar is the 50 open issues here, the 48 in igv.js, etc.   This  has not been prioritized.   As a workaround could I suggest you not load empty tracks?  

It always helps to have a read-made test case in a zip archive with instructions,  that usually takes more time than the fix.
